tdrs-backend/tdpservice/settings/common.py

```diff
@@ -281,10 +281,12 @@ class Common(Configuration):
     )
 
     # Sessions
-    SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
+    SESSION_ENGINE = "tdpservice.core.custom_session_engine"
+    SIGNED_COOKIE_EXPIRES = 60 * 60 * 24  # 24 hours
```

Comment:
@raftmsohani is this set to expire 24 hours from the initial session? if so, i wonder if it would be better to shorten this to 12 hours, which is on the very high end of an average work day. i'm also reaching out to our security officer about this.

Reply:
evidence below suggests expiry 24 hours into future. 

![Screenshot 2024-10-10 172400](https://github.com/user-attachments/assets/1caf1fc4-e023-471d-9805-feb39c9391c1)

Reply:
@raftmsohani TDP's ISSO's feedback: 

- this change is related to [AC-2-5](https://github.com/raft-tech/TANF-app/blob/develop/docs/Security-Compliance/Security-Controls/ac-2-5/index.md). We'll need to update this documentation as part of this work to capture how the session refresh/expiration works for ACF users. 
- let's also reduce the expiration to 12 hours
- we'll likely need to also complete risk-based decision documentation when the security team provides it.

Reply:
@ADPennington we shouldn't need to update the AC-2-5, since we are still following same time out concept. In fact, this PR makes sure we are exactly following this time out setting.
The only difference is: with new changes, even if the user is active, system will log the user out after 12hrs

Reply:
confirming that the AC-2-5 seems aligned with the behavior in this PR, particularly this part: `If the user chooses to extend the session, TDP will refresh the session.`